docker_build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ $# -ne 1 ]
+  then
+    echo "No board supplied, run as ./docker_build.sh <board name> or ./docker_build.sh list"
+    exit 1
+fi
+
+BOARD=$1
+
+git submodule update --init --recursive
+
+docker build . -t ardupilot
+docker run --rm -it -v "$(pwd)":/ardupilot ardupilot:latest ./waf configure --board="$BOARD"
```

Comment:
something is wrong with this invocation as the dockerfile provided is definitively able to compile all boards

Reply:
and the docker_build.sh  should be in Tools/scripts

Reply:
Hmm, very odd... I added this because people usually want to compile a single board, but that can be done when the board argument is omitted.

Reply:
Humm, I wonder if the mistake isn't just a path thing. Normally, the stm toolchain add to the .bashrc, so the mistake may be that doing that way the .bashrc isn't loaded

Reply:
That's very likely, I'll investigate.